Document Slack agent reconnect and one-org install (#5695)
mintlify/docs: agent/slack.mdx

    <img src="/images/agent/install-apps-dark.png" alt="The App settings section with the Slack and Notion install buttons." className="hidden dark:block" />
  </Frame>
3. Follow the Slack prompts to install the `mintlify` app in your workspace and grant the requested permissions.
4. Confirm the connection in your Mintlify dashboard to your Slack workspace.
5. Test that the agent is working and responds when you:
   - Send a direct message to the agent.
   - Mention the agent with `@mintlify` in a channel.

<Note>
  Each Slack workspace can connect to only one Mintlify organization at a time. If you try to install the agent into a workspace that is already connected to a different organization, the installation fails with a "wrong organization" error. Uninstall the app from the other organization first, or use a different Slack workspace.
</Note>

### Reconnect or reinstall the agent

If you must reauthorize the agent—for example, to grant new permissions after a Slack scope update or to recover from a revoked token—reinstall the app from your Mintlify dashboard.

> When you reinstall the agent into a workspace that is already connected to your organization, Mintlify refreshes the stored bot token in place. You skip the binding confirmation step and your existing channels, threads, and per-user integrations remain intact.

Reviewer: ```suggestion
When you reinstall the agent into a workspace that is already connected to your organization, Mintlify refreshes the stored bot token in place. Your existing channels, threads, and per-user integrations remain intact.
```

    <img src="/images/agent/install-apps-light.png" alt="The App settings section with the Slack and Notion install buttons." className="block dark:hidden" />
    <img src="/images/agent/install-apps-dark.png" alt="The App settings section with the Slack and Notion install buttons." className="hidden dark:block" />
  </Frame>
3. Follow the Slack prompts to install the `mintlify` app in your workspace and grant the requested permissions.

> 4. Confirm the connection in your Mintlify dashboard to bind the Slack workspace to your organization.

Reviewer: ```suggestion
4. Confirm the connection in your Mintlify dashboard to your Slack workspace.
```

5. Test that the agent is working and responds when you:
   - Send a direct message to the agent.
   - Mention the agent with `@mintlify` in a channel.

<Note>

>   Each Slack workspace can be connected to only one Mintlify organization at a time. If you try to install the agent into a workspace that is already connected to a different organization, the install fails with a "wrong organization" error. Uninstall the app from the other organization first, or use a different Slack workspace.

Reviewer: ```suggestion
  Each Slack workspace can connect to only one Mintlify organization at a time. If you try to install the agent into a workspace that is already connected to a different organization, the installation fails with a "wrong organization" error. Uninstall the app from the other organization first, or use a different Slack workspace.
```

</Note>

### Reconnect or reinstall the agent

> If you need to reauthorize the agent—for example, to grant new permissions after a Slack scope update or to recover from a revoked token—reinstall the app from the dashboard.

Reviewer: ```suggestion
If you must reauthorize the agent—for example, to grant new permissions after a Slack scope update or to recover from a revoked token—reinstall the app from your Mintlify dashboard.
```

1. Go to the [Agent](https://dashboard.mintlify.com/products/agent) page in your dashboard.
2. Click **Install Slack app** again.
3. Approve the Slack consent screen for the same workspace.

> 

Reviewer: ```suggestion
```
